Packs/IronDefense/pack_metadata.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "IronDefense",
-  "description": "The IronDefense Integration allows users to interact with IronDefense alerts within Demisto. The Integration provides the ability to rate alerts, update alert statuses, add comments to alerts, and to report observed bad activity.",
-  "support": "xsoar",
-  "currentVersion": "1.0.0",
-  "author": "Cortex XSOAR",
-  "url": "https://www.paloaltonetworks.com/cortex",
-  "email": "",
-  "created": "2020-04-14T00:00:00Z",
-  "categories": [
-    "Network Security"
-  ],
-  "tags": [],
-  "useCases": [],
-  "keywords": []
-}
+    "name": "IronDefense",
+    "description": "The IronDefense Integration allows users to interact with IronDefense alerts within Demisto. The Integration provides the ability to rate alerts, update alert statuses, add comments to alerts, and to report observed bad activity.",
+    "support": "xsoar",
```

Comment:
could you update the support type to `partner` please?
and the url/email to contact you for any problems with the integration(Your users who use you with demisto will be able to contact with these)

Reply:
I see, yes I'll make that update